src/components/SimpleRope.js

```diff
@@ -0,0 +1,25 @@
+import { SimpleRope as PixiSimpleRope } from 'pixi.js'
```

Comment:
In Pixi.js V5: Rope has move to SimpleRope

So I renamed Rope to SimpleRopehere.